### ISSUE TITLE
Docker: failed to start: Permission denied @ rb_sysopen - /root/.config/oxidized/log

### ISSUE THREAD

**Issue by a1ad** [ytti/oxidized#2664]
Just pulled the new docker image and I can't start oxidized anymore:

```
Permission denied @ rb_sysopen - /root/.config/oxidized/log
oxidized  | /usr/lib/ruby/2.7.0/logger/log_device.rb:95:in `initialize': Permission denied @ rb_sysopen - /root/.config/oxidized/log (Errno::EACCES)
oxidized  | 	from /usr/lib/ruby/2.7.0/logger/log_device.rb:95:in `open'
oxidized  | 	from /usr/lib/ruby/2.7.0/logger/log_device.rb:95:in `open_logfile'
oxidized  | 	from /usr/lib/ruby/2.7.0/logger/log_device.rb:86:in `set_dev'
oxidized  | 	from /usr/lib/ruby/2.7.0/logger/log_device.rb:18:in `initialize'
oxidized  | 	from /usr/lib/ruby/2.7.0/logger.rb:389:in `new'
oxidized  | 	from /usr/lib/ruby/2.7.0/logger.rb:389:in `initialize'
oxidized  | 	from /var/lib/gems/2.7.0/gems/oxidized-0.28.0/lib/oxidized.rb:46:in `new'
oxidized  | 	from /var/lib/gems/2.7.0/gems/oxidized-0.28.0/lib/oxidized.rb:46:in `setup_logger'
oxidized  | 	from /var/lib/gems/2.7.0/gems/oxidized-0.28.0/lib/oxidized/cli.rb:26:in `initialize'
oxidized  | 	from /var/lib/gems/2.7.0/gems/oxidized-0.28.0/bin/oxidized:8:in `new'
oxidized  | 	from /var/lib/gems/2.7.0/gems/oxidized-0.28.0/bin/oxidized:8:in `<top (required)>'
oxidized  | 	from /usr/local/bin/oxidized:23:in `load'
oxidized  | 	from /usr/local/bin/oxidized:23:in `<main>'
```

**** SEE FINAL EDIT FOR THE SOLUTION.

The container is starting in non root mode and the config files path is changed from /root/.config/oxidized/ to /home/oxidized/.config/oxidized according to the change here: https://github.com/ytti/oxidized/pull/2657/files

So why is oxidized trying to open the log file in /root/.. instead of /home/oxidized/...

edit: hmm i am not using volumes.  I have been mounting the storage like this:

```
    volumes:
      - ./config:/home/oxidized/.config/oxidized

```

Edit2:
The Readme is telling me i need to set up volumes like this:
    
```
volumes:
       - config:/root/.config/oxidized/config
       - router.db:/root/.config/oxidized/router.db
       - model:/root/.config/oxidized/model
       # if git is use as input
       - data:/root/.config/oxidized/backupcfg/
```

But it is lacking the volumes key in the example compose file.
volumes:
  config:
  router.db:
  model:

And how do you even configure a router.db file with the example given. I am confused. 

edit3:
Changing latest tag to 0.25.1 and my config is working again.

Please advise

FINAL EDIT:

I am stupid... i needed to change the file locations in my config file to the new oxidized home dir of course!!

So to summarize, the new docker image is running in non root mode in the folder: /home/oxidized/.config/oxidized
You need to change the config file dir in the compose file like this:
      - ./config:/root/.config/oxidized >  - ./config:/home/oxidized/.config/oxidized

AND do not forget to change all the locations in the config file /root/.config/oxidized/... to /home/oxidized/.config/oxidized/... like the log, pid, crash, output file, source file, ...

**Comment by a1ad** [comment]
Closed. See final edit.
